act-rules-format.bs

```diff
@@ -101,132 +105,146 @@ An ACT Rule MUST have a unique identifier that can be any unique text, such as p
+          <li>Any inapplicable outcomes: satisfied</li>
         </ul></li>
       </ul>
     </li>
   </ul></blockquote>
 </div>
 
-If the `failed` outcome can not be mapped to *not satisfied* for an accessibility requirement, that requirement MUST NOT be included in the accessibility requirements mapping. The optional [Background section](#background) could be used to list accessibility requirements and standards that may be thematically related, but for which the rule is not a failure test.
 
-If the rule does not map to any [=accessibility requirement=], the accessibility requirement mapping will only contain the explainer that it is not required for conformance to the [=accessibility requirements document=]. This is common with atomic rules used in composite rules, where accessibility requirements are only not satisfied with multiple atomic rules have the outcome`failed`.
+### Rules Without Accessibility Requirements ### {#rules-without-accessibility-requirements}
+
+If the rule does not map to any [=accessibility requirement=], the accessibility requirement mapping will only contain the explainer that it is not required for conformance to the [=accessibility requirements document=]. This is common with atomic rules used in composite rules.
 
 <div class=example>
-  <p><strong>Example:</strong> An ACT Rule that tests a best practice not included in any accessibility standard:</p>
+  <p>**Example:** An ACT Rule for WCAG 2.1 that tests if landmarks are used to satisfy [WCAG 2.1, success criterion 2.4.1 bypass blocks](https://www.w3.org/TR/WCAG21/#bypass-blocks):</p>
   <blockquote>
-    <p>This rule is <strong>not required</strong> for conformance to WCAG 2.0 or WCAG 2.1 at any level.</p>
+    <p>This rule is **not required** for conformance to WCAG 2.0 or WCAG 2.1 at any level.</p>
   </blockquote>
 </div>
 
-<div class=note>
-**Note**: While rules are often designed for one, or possibly a small collection of [=accessibility requirements document=], it is likely that other accessibility requirements documents also map to those rules. For example, rules may be written for WCAG 2.1, but many of those may map to a company's internal accessibility policy. In such a scenario, an external accessibility requirements mapping could be created. An external accessibility requirements mapping ammend the accessibility requirements mapping of a rule by adding mapping to a different accessibility requirements document. An external accessibility requirements mapping avoids duplication of a rule, for the sole purpose of changing the mapping.
-</div>
+If the `failed` outcome cannot be mapped to [=not satisfied=] for an [=accessibility requirement=], that requirement MUST NOT be included in the accessibility requirements mapping. The optional [Background section](#background) could be used to list accessibility requirements and standards when they are thematically related, but for which the rule is not a failure test.
```

Comment:
Suggest "If the failed outcome cannot be mapped to an accessibility requirement, there MUST NOT be an accessibility requirement in the accessibility requirement mapping.

Reply:
Or suggest making it positive "Only the accessibility requirements that map to the failed outcome must be included in the accessibility requirements mapping."